src/SipHash.chpl

```diff
@@ -69,20 +81,32 @@ module SipHash {
     return c;
   }
   
-  proc sipHash64(msg: [] uint(8), D): uint(64) {
+  proc sipHash64(msg: [] ?t, D): uint(64) {
     var (res,_) = computeSipHashLocalized(msg, D, 8);
     return res;
   }
 
-  proc sipHash128(msg: [] uint(8), D): 2*uint(64) {
+  proc sipHash128(msg: [] ?t, D): 2*uint(64) {
     return computeSipHashLocalized(msg, D, 16);
   }
 
-  private proc computeSipHashLocalized(msg: [] uint(8), D, param outlen: int) {
+  /*
+   * Compute hash of a single value
+   */
+  proc sipHash64(in val): uint(64) {
+    var (res,_) = computeSipHash(c_ptrTo(val), 0..#1, 8, 8);
+    return res;
+  }
+  
+  proc sipHash128(in val): 2*uint(64) {
+    return computeSipHash(c_ptrTo(val), 0..#1, 16, 8);
+  }
```

Comment:
Are the single value hash versions used in arkouda code, or are they just used in tests?

Reply:
Yes, these are used in the element-wise hashing functionality added in this PR, e.g. in

https://github.com/Bears-R-Us/arkouda/blob/32d18ef7ed165903722a591540e5237c93d9362b/src/EfuncMsg.chpl#L97-L99

The multi-value hash versions for `int(64)` and `real(64)` will be used in a future PR to implement a `GroupBy.hash(x)` method that hashes groups of values.